libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -192,9 +192,9 @@ struct CopyTranslate: public yul::ASTCopier
 			solAssert(false, "");
 
 		if (isdigit(value.front()))
-			return yul::Literal{_identifier.location, yul::LiteralKind::Number, yul::YulString{value}, {}};
+			return yul::Literal{_identifier.debugData, yul::LiteralKind::Number, yul::YulString{value}, {}};
```

Comment:
Do we actually have any useful location here? It is fine though, as I guess this is a good example where the `originLocation` will be useful to be kept.

Reply:
What's the difference between `location` and `originLocation`?

Reply:
In case when we run yul in `irMode` we will have origin location to be location from solidity code, and location would be actual location of in ir(yul)